src/main/resources/InvoicesCode/InvoicesTranslations.fr.xml

```diff
@@ -100,12 +100,12 @@ invoices.livetable.type=Type
 invoices.livetable.firstProduct=Produit/Service
 
 # UI
-invoices.description=Application pour gérer et créer des factures (ou des remboursements). Elle produit aussi des notifications par mail lors de la création de nouvelles factures.
+invoices.description=Application pour gérer et créer des factures (ou des remboursements). Elle produit aussi des notifications par courriel lors de la création de nouvelles factures.
 invoices.actions=Actions
 invoices.actions.invoice=Nouvelle facture
 invoices.actions.invoice.empty=Vide
-invoices.actions.reimbursement=Nouveau remboursement
-invoices.actions.reimbursement.empty=Vide
+invoices.actions.reimboursement=Nouveau remboursement
```

Comment:
This is affecting the translation key, not the value.

Reply:
This is because the key is wrong in the default version. We need to wait for weblate to update it now.